frontend/web/components/pages/onboarding/InlineInput/InlineInput.scss

```
@@ -1,32 +1,24 @@
-// Inline editable value in the welcome sentence: an action underline + pencil,
-// with a faint highlight on hover/focus. Rationale lives in InlineInput.tsx.
 .inline-input {
   display: inline-flex;
   align-items: center;
   vertical-align: baseline;
   gap: 4px;
-  padding: 0 4px;
-  border-radius: var(--radius-xs);
+  padding: 1px 8px;
+  // Round the top only; the bottom stays square so it sits flush with the
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)